experiment/gerrit-onboarder/onboarder.go

```diff
@@ -0,0 +1,418 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"math/rand"
+	"os"
+	"regexp"
+	"strings"
+	"time"
+
+	"github.com/sirupsen/logrus"
+
+	"k8s.io/test-infra/prow/cmd/generic-autobumper/bumper"
+)
+
+const (
+	uuID              = 0
+	groupName         = 1
+	groupsFile        = "groups"
+	projectConfigFile = "project.config"
+
+	accessHeader          = `[access "refs/*"]`
+	prowReadAccessFormat  = "read = group %s"
+	prowLabelAccessFormat = "label-Verified = -1..+1 group %s"
+	labelHeader           = `[label "Verified"]`
+	labelEquals           = "label-Verified ="
+
+	accessRefsRegex = `^\[access "refs\/.*"\]`
```

Comment:
`regexp.MustCompile` this as global var to avoid runtime regexp error

Reply:
Done